### DISCUSSION TITLE
Host a PDF in a website?

### DISCUSSION THREAD

**Discussion by fsolt** [quarto-dev/quarto-cli#790] [Q&A] [answered]
On my website, which I'm now rebuilding in Quarto, I'd like to be able to host a few of my own preprints that are in PDF format.  I've found that leaving a PDF in a directory within the website's repo before rendering doesn't get a copy into the directory within the YAML-specified `output-dir`, although after rendering, I can copy these PDFs into `output-dir` and links to them will work fine.  But re-rendering deletes these files, so they need copied over again.  Is there a way to indicate that a website should include these PDFs?

**Comment by jjallaire** [comment, marked as answer]
You need to do one of two things:

1) Link to the PDFs somewhere in your HTML and they should be automatically copied; or
2) List the PDFs as site `resources` (as described here: https://quarto.org/docs/websites/website-tools.html#site-resources)

**Comment by fsolt** [comment]
1. wasn't doing the trick for some reason, but 2. worked like a charm!  Thanks so much for your help, J.J., and for all your work on this!
